src/contrib/PACKAGES

```diff
@@ -26,6 +26,6 @@ NeedsCompilation: no
 Package: rzmq
 Version: 0.7.7
 License: GPL-3
-MD5sum: 297240be35b02cb04ce378f31083d67d
+MD5sum: 4a0114038502b470f7e62cd48ec89934
```

Comment:
OK, the new hash is the one that's in the repo, so I'll merge this. Thanks for fixing it quickly!